app/components/forms/wizard/basic-details-step.js

```diff
@@ -141,7 +141,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
       return parseInt($('.ui.form').form('get value', 'ticket_min_order'), 10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'), 10);
     };
-
+    $.fn.form.settings.rules.checkValidTimeDifference = () => {
+      return moment($("[name=start_time]")[0].value , 'HH:mm').isBefore(moment($("[name=end_time]")[0].value, 'HH:mm'));
```

Comment:
```suggestion
      return moment($('[name=start_time]')[0].value, 'HH:mm').isBefore(moment($("[name=end_time]")[0].value, 'HH:mm'));
```

Reply:
@sachinchauhan2889  should i have to add a line break at the poin you sujjested?